app/src/main/java/chat/rocket/android/activity/LoginContract.java

```diff
@@ -0,0 +1,17 @@
+package chat.rocket.android.activity;
+
+import chat.rocket.android.shared.BaseContract;
+
+public interface LoginContract {
+
+  interface View extends BaseContract.View {
```

Comment:
The name `View` is a little confusing. It is nothing to do with `android.view.View`.

`Behavior` or `Action`  would be better.